nexus/internal/filetransfer/file_transfer_default.go

```diff
@@ -65,10 +66,21 @@ func (ft *DefaultFileTransfer) Upload(task *UploadTask) error {
 	return nil
 }
 
+func (d *DownloadTask) ensureDownloadRootDir() error {
+	baseDir := path.Dir(d.Path)
+	info, err := os.Stat(baseDir)
+	if err == nil && info.IsDir() {
+		return nil
+	}
```

Comment:
Nit: I don't think this is necessary.

Reply:
I think `os.MkdirAll()` will return an error if we try to make a directory that already exists...